infer/src/pulse/PulseFormula.ml

```diff
@@ -4052,7 +4167,51 @@ module Intervals = struct
                  let+ formula = incorporate_new_eqs new_eqs {formula with phi} in
                  (formula, new_eqs) )
         in
-        refine v1_opt i1_better_opt (formula, new_eqs) >>= refine v2_opt i2_better_opt
+        let binop_unknown (binop : Binop.t) =
```

Comment:
What is the purpose of this new code? This is a somewhat trivial subdomain of the formula so this change is a bit surprising.